scripts/error_code_generator_defs/result_codes.json

```diff
@@ -652,6 +652,14 @@
               {
                 "name": "ADUC_ERC_CONTENT_DOWNLOADER_UNSUPPORTED_CONTRACT_VERSION",
                 "value": 13
+              },
+              {
+                "name": "ADUC_ERC_CONTENT_DOWNLOADER_ONDOWNLOADPROC_NOTIMP",
+                "value": 14
+              },
+              {
+                "name": "ADUC_ERC_CONTENT_DOWNLOADER_ONDOWNLOADENDPROC_NOTIMP",
```

Comment:
is there no ERRNO for NOTIMPLEMENTED?  I have a bug on this where our codes aren't generic, and so we end up with ONDOWNLOADPROC_NOTIMP, ONDOWNLOADPROCEND_NOTIMPL.  What happens if we add more functions that support NOTIMP, do we add FOO_NOTIMPL, BAR_NOTIMPL?

[Bug 43012743](https://microsoft.visualstudio.com/OS/_workitems/edit/43012743): Error codes are TOO content handler specific.

(marked as 2303, but I haven't seen progress)